src/main/java/org/dita/dost/invoker/Main.java

```diff
@@ -438,23 +440,35 @@ private void processArgs(final String[] arguments) {
   }
 
   private Map<String, Object> getLocalProperties(File ditaDir) {
-    final File localPropertiesFile = new File(ditaDir, "local.properties");
-    if (localPropertiesFile.exists()) {
-      if (args.msgOutputLevel >= Project.MSG_VERBOSE) {
-        System.out.println("Reading  " + localPropertiesFile);
-      }
-      try (InputStream in = Files.newInputStream(localPropertiesFile.toPath())) {
-        final Properties localProperties = new Properties();
-        localProperties.load(in);
-        return localProperties
-          .entrySet()
-          .stream()
-          .collect(Collectors.toMap(e -> e.getKey().toString(), Map.Entry::getValue));
-      } catch (IOException e) {
-        System.err.println("Failed to read " + localPropertiesFile);
-      }
+    final Map<String, Object> res = new HashMap<>();
+    Stream
+      .of(
+        new File(ditaDir, CONFIGURATION_FILE_OLD),
```

Comment:
The priority order here is reversed, because we use `putAll` to overwrite existing values.